src/Auth0Client.ts

```diff
@@ -558,9 +558,14 @@ export default class Auth0Client {
    * with `getTokenSilently` is that this doesn't return a token, but it will
    * pre-fill the token cache.
    *
+   * This method also heeds the `auth0.is.authenticated` cookie, as an optimization
+   *  to prevent calling Auth0 unnecessarily. If the cookie is not present because
+   * there was no previous login (or it has expired) then tokens will not be refreshed.
+   *
    * It should be used for silently logging in the user when you instantiate the
    * `Auth0Client` constructor. You should not need this if you are using the
-   * `createAuth0Client` factory.
+   * `createAuth0Client` factory. If you are using the `Auth0Client` constructor
+   * directly, you can use `checkSession` immediately afterwards to check login state.
```

Comment:
This paragraph speaks about:

- using constructor
- not using constructor
- using constructor

I think it reads a bit weird. Could we keep the parts about using the constructor together?

Reply:
Good point. Actually I think we can just drop this second part.